pyblazing/pyblazing/apiv2/context.py

```diff
@@ -584,36 +677,35 @@ def adjust_due_to_missing_rowgroups(metadata, files):
 
     missing_file_inds = list(reversed(missing_file_inds))
     for ind in missing_file_inds:
-        mask = metadata['file_handle_index'] > ind
-        metadata['file_handle_index'][mask] = metadata['file_handle_index'][mask] - 1
+        mask = metadata["file_handle_index"] > ind
+        metadata["file_handle_index"][mask] = metadata["file_handle_index"][mask] - 1
     return metadata, new_files
 
 
 def distributed_initialize_server_directory(client, dir_path):
 
-    # lets make host_list which is a list of all the unique hosts. 
+    # lets make host_list which is a list of all the unique hosts.
     # This way we do the logging folder creation only once per host (server)
-    host_list = list(set([value['host'] for key,value in client.scheduler_info()["workers"].items()]))
+    all_items = client.scheduler_info()["workers"].items()
+    host_list = list(set([value["host"] for key, value in all_items]))
     initialized = {}
     for host in host_list:
-        initialized[host]=False
+        initialized[host] = False
 
     dask_futures = []
     for worker, worker_info in client.scheduler_info()["workers"].items():
-        if not initialized[worker_info['host']]:
+        if not initialized[worker_info["host"]]:
             dask_futures.append(
-                client.submit(
-                    initialize_server_directory,
-                    dir_path,
-                    workers=[worker]))
-            initialized[worker_info['host']] = True
-    
-    for connection in dask_futures:
-        made_dir = connection.result()
-    
+                client.submit(initialize_server_directory, dir_path, workers=[worker])
+            )
+            initialized[worker_info["host"]] = True
+
+    # for connection in dask_futures:
```

Comment:
Esto no creo que deberia estar commentado. Aqui aunque no se esta haciendo nada con `made_dir` queremos esperar a que retornen los submit. Podriamos recolectar todos los `made_dir` y validar que todos retornan true

Reply:
ok

Reply:
Now this happen:
pyblazing/pyblazing/apiv2/context.py:700:9: F841 local variable 'made_dir' is assigned to but never used